ibu-imager/postpivot/postpivot.go

```diff
@@ -92,6 +92,10 @@ func (p *PostPivot) PostPivotConfiguration(ctx context.Context) error {
 		return err
 	}
 
+	if err := utils.RunOnce("set_cluster_id", p.workingDir, p.log, p.setNewClusterID, ctx, client, clusterInfo); err != nil {
```

Comment:
Can this be done by recert at the same time it's handling the cluster naming? Right now, there's a window where the API is available, but the clusterID is that of the seed. @omertuc ?
In the meantime, maybe this can be the first change applied once the API is available